ProductCatalog unit tests failing

```
# github.com/GoogleCloudPlatform/microservices-demo/src/productcatalogservice [github.com/GoogleCloudPlatform/microservices-demo/src/productcatalogservice.test]
./server_test.go:32:14: cannot use 0 (type untyped int) as type string in argument to run
FAIL	github.com/GoogleCloudPlatform/microservices-demo/src/productcatalogservice [build failed]
```

--- a/src/productcatalogservice/server_test.go
+++ b/src/productcatalogservice/server_test.go
@@ -29,7 +29,7 @@ import (
 
 func TestServer(t *testing.T) {
 	ctx := context.Background()
-	addr := run(0)
+	addr := run(port)
 	conn, err := grpc.Dial(addr,
 		grpc.WithInsecure(),
 		grpc.WithStatsHandler(&ocgrpc.ClientHandler{}))

